samila/functions.py

```diff
@@ -117,15 +119,15 @@ def nft_storage_upload(api_key, data):
 
 def save_data_file(data1, data2, matplotlib_version, file_adr):
     """
-    Save config as file.
+    Save data as file.
 
     :param data1: data 1
     :type data1: list
     :param data2: data 2
     :type data2: list
     :param matplotlib_version: matplotlib version
     :type matplotlib_version: str
-    :param file_adr: file address
+    :param file_adr: file addresses
```

Comment:
Why `addresses`?

Reply:
Good point.
Fixed in https://github.com/sepandhaghighi/samila/pull/71/commits/d76252c5d3cbbf5e6447fbd120e5b27b2b4702cf